packages/functions/src/donor/BookAppointmentHandler.ts

```diff
@@ -2,16 +2,10 @@ import { getDonorOrThrow, setDonor } from "../dal/DonorDataAccessLayer";
 import {
   getAppointmentsByDonorIdInTime,
   getAppointmentsByIds,
-  setAppointment,
+  setAppointment
 } from "../dal/AppointmentDataAccessLayer";
 import * as admin from "firebase-admin";
-import {
-  AppointmentStatus,
-  BookingChange,
-  DbDonor,
-  FunctionsApi,
-  Hospital,
-} from "@zm-blood-components/common";
+import { AppointmentStatus, BookingChange, DbDonor, FunctionsApi, Hospital } from "@zm-blood-components/common";
```

Comment:
This means you ran install in `function` folder and not in the outer one.

run `yarn install` in the root folder, and then `style` again